onewire_bus/src/onewire_crc.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-FileCopyrightText: 2022-2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "onewire_crc.h"
+
+#define FAST_CRC 1 // define this to use the fast CRC table
```

Comment:
Do we expect users to change this to '0' often?

Reply:
I think in most cases, users won't touch this. Another implementation was added for my understanding of its underlying CRC. :smiley:

Reply:
OK, because modifying sources inside `managed_components` dir leads to warnings. Let's keep it as it is for now